### DISCUSSION TITLE
打开动态刷新率后，gpu占用异常偏高

### DISCUSSION THREAD

**Discussion by JiayanJohnnyChu** [ywmoyue/biliuwp-lite#259] [General]
使用surface pro 8在打开动态刷新率的情况下，观看直播时，gpu占用极高，关闭动态刷新率即恢复正常

**Comment by ywmoyue** [comment]
没懂，动态刷新率是在哪开

**Comment by JiayanJohnnyChu** [comment]
> 没懂，动态刷新率是在哪开

win11系统设置里面的一个显示设置，系统  >显示>高级显示器。

![image](https://github.com/ywmoyue/biliuwp-lite/assets/52626624/9b1d9ac9-eca3-4284-9e50-3123aa2fbc2f)

**Comment by ywmoyue** [comment]
可能是surface pro8特有，我这没这选项
![image](https://github.com/ywmoyue/biliuwp-lite/assets/28388300/16294101-d432-47da-9ef6-074c7e713e64)

**Comment by JiayanJohnnyChu** [comment]
> 可能是surface pro8特有，我这没这选项
> 
> ![image](https://github.com/ywmoyue/biliuwp-lite/assets/28388300/16294101-d432-47da-9ef6-074c7e713e64)
> 
> 

好像是需要支持可变刷新率的显示器和WDDM3.1驱动。
不知道这个gpu占用过高是app的bug还是系统的bug。
